docs/remote/containers-advanced.md

```diff
@@ -162,43 +202,56 @@ To connect to both:
 
-### Accessing Docker Remotely
+### Using SSH to connect to remote Docker host
 
-If you have a remote [Docker Machine](https://docs.docker.com/machine/overview/) running on a remote server, you can work with it from your local machine using the Docker CLI by specifying [environment variables like `DOCKER_HOST`, `DOCKER_CERT_PATH`, `DOCKER_TLS_VERIFY`](https://docs.docker.com/machine/reference/env/). However, this port is often not exposed publicly since it can leave the machine vulnerable if not secured properly. (For example, installing Docker CE / Desktop does not open up a port by default.)
+You can use the Docker CLI locally with a remote [Docker Machine](https://docs.docker.com/machine/overview/) by setting [local environment variables like `DOCKER_HOST`, `DOCKER_CERT_PATH`, `DOCKER_TLS_VERIFY`](https://docs.docker.com/machine/reference/env/). Since VS Code uses the Docker CLI under the hood, you can use these same environment variables to connect the Remote - Containers extension to this same remote host.
 
-A more secure way is to use an SSH tunnel to access the Docker socket as needed. If you have an [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) installed, you can run the following commands in a local terminal / command prompt to start up VS Code so that it connects to the remote SSH host. Replace `user@hostname` with the appropriate remote user and hostname / IP for your server and ensure `code-insiders` is in your path.
+```bash
+export DOCKER_HOST=your-remote-machine-fqdn-or-ip-here:2375
+# Or on Windows: SET DOCKER_HOST=your-remote-machine-fqdn-or-ip-here:2375
+code-insiders
+```
 
-On **macOS or Linux**:
+Once set, you can use VS Code to [attach to any running container](/docs/remote/containers.md#attaching-to-running-containers) on the remote host or [use specialized, local `devcontainer.json` files to create / connect to a remote dev container](#using-devcontainerjson-to-work-with-a-remote-dev-container).
+
+However, Docker CE / Desktop will not expose the required Docker daemon TCP port by default since it can leave the machine vulnerable if not secured properly. The Docker CLI uses a local Unix socket (or named pipe on Windows) to communicate instead.
+
+Fortunately, you can use a SSH tunnel to forward a local socket to your local machine on an needed basis. If you have an [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) installed and `code-insiders` in your path, you can run the following commands in a local terminal / command prompt to connect VS Code to the remote Docker Machine.
+
+On **macOS or Linux**, run the following replacing `user@hostname` with the remote user and hostname / IP for your server:
 
 ```bash
 export DOCKER_HOST=localhost:23750
 code-insiders
 ssh -NL localhost:23750:/var/run/docker.sock user@hostname
```

Comment:
Good stuff, didn't know this works.

Reply:
Yeah I never use docker machine since this is an option.  Safer and simpler to setup.